src/std/fs.ab

```diff
@@ -1,55 +1,71 @@
+import * from "std/env"
+
+/// Check if directory exists
 pub fun dir_exist(path) {
     $[ -d "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Check if file exists
 pub fun file_exist(path) {
     $[ -f "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Get the file content
 pub fun file_read(path) {
     return $< "{path}"$?
 }
 
+/// Write the content to the file
+/// Doesn't check if the file exist
 pub fun file_write(path, content) {
     return $echo "{content}" > "{path}"$?
 }
 
+/// Append the content to the file
+/// Doesn't check if the file exist
 pub fun file_append(path, content) {
     return $echo "{content}" >> "{path}"$?
 }
 
+/// Create a symbolic link
+/// If the file doens't exist return a boolean and print a message
 pub fun create_symbolic_link(origin: Text, destination: Text): Bool {
     if file_exist(origin) {
         unsafe $ln -s "{origin}" "{destination}"$
         return true
     }
 
-    echo "The file {origin} doesn't exist!"
+    error("The file {origin} doesn't exist!")
     return false
 }
 
+/// Create a directory with all intermediate directories as required
 pub fun create_dir(path: Text): Null {
     if not dir_exist(path) {
         unsafe $mkdir -p "{path}"$
     }
 }
 
+/// Set the file as executable
+/// If the file doesn't exist return a boolean and print a message
 pub fun make_executable(path: Text): Bool {
     if file_exist(path) {
         unsafe $chmod +x "{path}"$
         return true
     }
 
-    echo "The file {path} doesn't exist!"
+    error("The file {path} doesn't exist!")
```

Comment:
I think this is not necessary if the function return false.
maybe is not an error if is an user input.
```
pub fun make_executable(path: Text): Bool {
  silent $chmod +x "{path}"$ failed {
          return false
  }    
  return true
}
```

Reply:
also this "resolve" circular import problem

Reply:
I don't like silent as we can't be sure if it is executed and usually a check avoid a bash error in this case generated by chmod.
I think that is part of the discussion we have in the dedicated discussion about the error reporting.

About the circular I opened a ticket as it shouldn't happen as I am just importing libraries.

Reply:
maybe for this PR we can keep the function the same. this introduced a breaking change or the function call must be : `error("The file {path} doesn't exist!", 0)`.
I think we can open discussion about error displaying and verbose level management.